client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.scss

```diff
@@ -20,6 +20,20 @@
 	}
 }
 
+.email-provider-stacked-card__title-price-badge {
+	.promo-card__price {
+		.price__cost {
+			.professional-email-card__interval, .professional-email-card__mailbox {
+				font-size: 0.875rem;
+				color: var( --studio-gray-80 );
+			}
+			.professional-email-card__sale {
+				color: var( --studio-green-50 );
+			}
+		}
+	}
+}
+
```

Comment:
These CSS selectors were removed by my changes to the `PriceWithInterval` component yesterday. I think we should remove this block entirely.